.github/workflows/run_jmeter_test.yaml

```diff
@@ -0,0 +1,35 @@
+name: Run JMeter Test
+
+on:
+  workflow_dispatch:
+    inputs:
+      branch_name:
+        description: 'The name of the branch to build from'
+        required: true
+
+      test_to_run:
+        description: 'The load test to run from load-test/'
+        type: choice
+        options:
+        - applicant_submit_application
+        - applicant_landing_page
+        - ti_landing_page
+
+jobs:
+  testing:
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.event.inputs.branch_name }}
+      - name: JMeter Test
+        uses: QAInsights/PerfAction@v5.6.2
+        with:
+          test-plan-path: load_test/${{ github.event.inputs.test_to_run }}.jmx
+          args: ""
```

Comment:
question (non-blocking): Do we need this if we aren't setting it?

Reply:
I think we do based on this example https://github.com/marketplace/actions/perfaction-for-jmeter#example-1-with-no-arguments?